src/main/java/org/mule/extension/file/common/api/FileSystem.java

```diff
@@ -114,21 +114,54 @@
    * This method also makes a best effort to determine the mime type of the file being read. a {@link MimetypesFileTypeMap} will
    * be used to make an educated guess on the file's mime type
    *
-   * @param config                the config that is parameterizing this operation
-   * @param filePath              the path of the file you want to read
-   * @param lock                  whether or not to lock the file
-   * @param timeBetweenSizeCheck  wait time between size checks to determine if a file is ready to be read in milliseconds.
+   * @param config               the config that is parameterizing this operation
+   * @param filePath             the path of the file you want to read
+   * @param lock                 whether or not to lock the file
+   * @param timeBetweenSizeCheck wait time between size checks to determine if a file is ready to be read in milliseconds.
    * @return An {@link Result} with an {@link InputStream} with the file's content as payload and a {@link FileAttributes} object
-   *         as {@link Message#getAttributes()}
+   * as {@link Message#getAttributes()}
    * @throws IllegalArgumentException if the file at the given path doesn't exist
    */
   default Result<InputStream, A> read(FileConnectorConfig config, String filePath, boolean lock, Long timeBetweenSizeCheck) {
-    return read(config, filePath, lock, timeBetweenSizeCheck);
+    return read(config, filePath, lock);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "read"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule-file-commons&pullRequest=56&issues=AWm2SxrdotIXnmsUPRcB&open=AWm2SxrdotIXnmsUPRcB)

[ISSUE_KEY]: AWm2SxrdotIXnmsUPRcB